### ISSUE TITLE
What triggered this interface in vc_inference?

### ISSUE THREAD

**Issue by treya-lin** [Plachtaa/VITS-fast-fine-tuning#417]
Hi I'm running vc_inference.py on linux server, and I found that it always pop out an interface showing something like "Use arrow keys to move blabla" I don't know what that interface is intended for.  

At first I thought it was due to the `webbrowser` library, so I editted the script like this, but that interface still exists.

```
142             btn = gr.Button("Convert!")
143             btn.click(vc_fn, inputs=[source_speaker, target_speaker, record_audio, upload_audio],
144                       outputs=[message_box, converted_audio])
145     # webbrowser.open("http://127.0.0.1:7860")
146     # app.launch(share=args.share)
147     app.launch(share=args.share, inbrowser=True, server_name="0.0.0.0",server_port=args.port)

```

 it looks like this:
![image](https://github.com/Plachtaa/VITS-fast-fine-tuning/assets/86940562/74776e5e-6432-4e06-a2e2-97cc0fffbabb)

For now I just type in q for quit and then y for yes, and the interface disappears and I can still open the demo webpage. But I do wanna know what is triggering this interface, what it is intended for and if  it is possible to mute it? Thanks!

**Comment by anfogy** [comment]
`app.launch(share=args.share, inbrowser=True, server_name="0.0.0.0",server_port=args.port)` is triggering the interface, but I'm not sure what does `mute the interface` mean, I would glad if you can provide more details.

**Comment by treya-lin** [comment]
> `app.launch(share=args.share, inbrowser=True, server_name="0.0.0.0",server_port=args.port)` is triggering the interface, but I'm not sure what does `mute the interface` mean, I would glad if you can provide more details.

So this interface appears on your local terminal too? I used gradio-based service in other projects too but I didn't see this interface before. Did you also just click q and y?

**Comment by anfogy** [comment]
> > `app.launch(share=args.share, inbrowser=True, server_name="0.0.0.0",server_port=args.port)` is triggering the interface, but I'm not sure what does `mute the interface` mean, I would glad if you can provide more details.
> 
> So this interface appears on your local terminal too? I used gradio-based service in other projects too but I didn't see this interface before. Did you also just click q and y?

no, it won't appear in local terminal, I dunno what happened but maybe try updating gradio versions?
